### PR TITLE
Created the frontend of the AttractionDetails page with the addition …

### PR DESCRIPTION
**Pull request by syasrebii**
…of the date selection functionality on the home page

I made a change to our home page by adding a "date" type for our return & departure dates. I also created the interface for our attractions details page.

### CODE REVIEW

File: travvy/src/components/AttractionDetails.vue [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
   <div>
-      <h1>Attraction Details for {{}} </h1>
+    <!-- Button to go to go back to the AttractionsList page --> 
+    <router-link to="/AttractionList" tag="button" class="back"><span>Back</span></router-link>
+
+    <h1>CN Tower </h1>
+    <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
+    <br>
+    <h1>Description</h1>
+    <p>The CN Tower is a 553.3 m-high concrete communications and observation tower located in Downtown Toronto, Ontario, Canada.</p>
```
Comment (rushanniazi) [inline travvy/src/components/AttractionDetails.vue:10]: I can see that youve hard coded this description for this instance, however when we are able to access the data from the database through the backend server, you are planning on changing this to be dynamic, correct?

File: travvy/src/components/AttractionDetails.vue [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+    <h3>Address:</h3>
+    <h3>Email:</h3>
+    <h3>Phone Number:</h3>
+    <h3>Ticket Pricing:</h3>
+    <h3>Maximum Group Size:</h3>
+    <h3>Are masks required:</h3>
```
Comment (rushanniazi) [inline travvy/src/components/AttractionDetails.vue:16]: We might want to add other attributes here as well such as start date, close date for the season in future iterations

File: travvy/src/components/AttractionDetails.vue [hunk 3]
```diff
@@ -29,4 +44,50 @@
...
+        rgba(255,242,225) 100%)
+}
+.back span {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  transition: 0.5s;
+}
+
+.back span:after {
```
Comment (rushanniazi) [inline travvy/src/components/AttractionDetails.vue:76]: I see you added button functionality/behavior when the user hovers, similar to the home page buttons!! nice touch :)

File: travvy/src/components/Home.vue [hunk 4]
```diff
@@ -24,9 +24,9 @@
     <!-- data binding dates, corresponding data object below -->
     <label for="Dates">     Select Departure Date:</label>
     <!-- user selects dates they are travelling in --> 
-      <input type="text" class="dates"> 
+      <input type="Date" class="dates">
```
Comment (rushanniazi) [inline travvy/src/components/Home.vue:27]: So great that you were able to convert these input boxes to be treated as date selectors!

File: travvy/src/components/AttractionDetails.vue (hunk 1 above)
Comment (syasrebii) [reply travvy/src/components/AttractionDetails.vue:10]: Good catch, Rushan! Yes, that's correct, for the time being, I'm just stubbing for testing purposes. After this iteration, we'd definitely want the data to be pulled directly from our database.

File: travvy/src/components/AttractionDetails.vue (hunk 2 above)
Comment (syasrebii) [reply travvy/src/components/AttractionDetails.vue:16]: Yeah, that's a good point, I'll definitely account for those once we add them in just to add more details about each attraction!

File: travvy/src/components/AttractionDetails.vue (hunk 1 above)
Comment (rushanniazi) [reply travvy/src/components/AttractionDetails.vue:10]: Ok great!

File: travvy/src/components/AttractionDetails.vue (hunk 2 above)
Comment (rushanniazi) [reply travvy/src/components/AttractionDetails.vue:16]: Sounds good!

File: travvy/src/components/AttractionDetails.vue (hunk 3 above)
Comment (syasrebii) [reply travvy/src/components/AttractionDetails.vue:76]: Thanks!! Just wanted to make it cohesive throughout the platform!! Also added a gradient for fun :)

**Comment by rushanniazi** [comment]
Everything looks good to me and no conflicts

**Pull request merged**